NodeApp/TodoList.js

```diff
@@ -0,0 +1,61 @@
+var express = require("express");
+const fs = require('fs');
+const zlib = require('zlib');
+var app = express();
+let idIndex = 5;
+
+// Method to show contents of the list to the user:
+app.get("/View", (req, res) => {
+    let jsonFile = fs.readFileSync('./MyFile.json');
+    let file = JSON.parse(jsonFile);
+    res.send(file);
+})
+
+// Method for adding new items to list:
+app.post("/Add", (req, res) => {
+    let jsonFile = fs.readFileSync('./MyFile.json');
```

Comment:
it is a good practice to move the file path to a constant since you'll be using it more later on and if you change it, you'll be required to change in several places.